pokedex/data/csv/moves.csv

```diff
@@ -825,6 +825,79 @@ id,identifier,generation_id,type_id,power,pp,accuracy,priority,target_id,damage_
 824,glacial-lance,8,15,130,5,100,0,11,2,1,,,,
 825,astral-barrage,8,8,120,5,100,0,11,3,1,,,,
 826,eerie-spell,8,14,80,5,100,0,10,3,1,,,,
+827,dire-claw,0,3,80,15,100,0,0,2,,,,,
+828,psyshield-bash,0,13,70,10,90,0,0,2,,,,,
+829,power-shift,0,0,0,10,,0,0,1,,,,,
+830,stone-axe,0,5,65,15,90,0,0,2,,,,,
+831,springtide-storm,0,17,100,5,80,0,0,3,,,,,
+832,mystical-power,0,13,70,10,90,0,0,3,,,,,
+833,raging-fury,0,9,120,10,100,0,0,2,,,,,
+834,wave-crash,0,10,120,10,100,0,0,2,,,,,
+835,chloroblast,0,11,150,5,95,0,0,3,,,,,
+836,mountain-gale,0,14,100,10,85,0,0,2,,,,,
+837,victory-dance,0,1,0,10,,0,0,1,,,,,
+838,headlong-rush,0,4,120,5,100,0,0,2,,,,,
+839,barb-barrage,0,3,60,10,100,0,0,2,,,,,
+840,esper-wing,0,13,80,10,100,0,0,3,,,,,
+841,bitter-malice,0,7,75,10,100,0,0,3,,,,,
+842,shelter,0,8,0,10,,0,0,1,,,,,
+843,triple-arrows,0,1,90,10,100,0,0,2,,,,,
+844,infernal-parade,0,7,60,15,100,0,0,3,,,,,
+845,ceaseless-edge,0,16,65,15,90,0,0,2,,,,,
+846,bleakwind-storm,0,2,100,10,80,0,0,3,,,,,
+847,wildbolt-storm,0,12,100,10,80,0,0,3,,,,,
+848,sandsear-storm,0,4,100,10,80,0,0,3,,,,,
+849,lunar-blessing,0,13,0,5,,0,0,1,,,,,
+851,tera-blast,0,0,80,10,100,0,0,3,,,,,
```

Comment:
850 Take Heart is missing in the original data.
https://bulbapedia.bulbagarden.net/wiki/Take_Heart_(move)